sysmonconfig-export-block.xml

```diff
@@ -1322,6 +1323,7 @@
 			<!-- Executables dropped to suspicious folders -->
 			<TargetFilename condition="begin with">C:\Users\Public\</TargetFilename> <!-- often used staging directories; could cause false positives --> 
 			<TargetFilename condition="begin with">C:\Perflogs\</TargetFilename> <!-- often used staging directories --> 
+			<TargetFilename condition="begin with">C:\Users\;\AppData\LocalLow\</TargetFilename> <!-- often used staging directories --> 
```

Comment:
Isn't LocalLow used by browser to store data downloaded from the Internet, e.g. Internet Explorer

Reply:
I wouldn't want to interfere with user downloads that are legitimate

Reply:
It shouldn't contain any exe to my knowledge as those should be dropped to local/temp but I could be wrong. I'll remove it then by precaution